src/index.ts

```diff
@@ -1,38 +1,48 @@
-/**
- * @fileoverview Setup Wasmer
- */
-
 //Imports
-import got from 'got';
-import {getInput, addPath, debug, endGroup, exportVariable, info, setFailed, startGroup} from '@actions/core';
-import {createWriteStream} from 'fs';
-import {exec} from '@actions/exec';
-import {homedir} from 'os';
-import {join} from 'path';
-import {path} from 'temp';
-import {promisify} from 'util';
-import {pipeline as pipelineCB} from 'stream';
+import got from "got";
+import {
+  getInput,
+  addPath,
+  debug,
+  endGroup,
+  exportVariable,
+  info,
+  setFailed,
+  startGroup,
+} from "@actions/core";
+import { createWriteStream } from "fs";
+import { exec } from "@actions/exec";
+import { homedir } from "os";
+import { join } from "path";
+import { path } from "temp";
+import { promisify } from "util";
+import { pipeline as pipelineCB } from "stream";
```

Comment:
I think you're only changing the indentation here. Everything except the last two import lines are the same.
Would be nice to have lint config in the repo.